src/app/modules/authentication/components/request-access-form/request-access-form.component.html

```diff
@@ -3,7 +3,7 @@
         <div class="request-access-card">
             <h1 class="request-access-title">Request Access</h1>
 
-            <form [formGroup]="requestAccessForm" (ngSubmit)="onSubmitRegistration()" class="request-access-form">
+            <form [formGroup]="requestAccessForm" (ngSubmit)="submit.focus(); onSubmitRegistration()" class="request-access-form">
```

Comment:
See [Email-async-validator](https://github.com/lpboucher/dak-coffee-wholesale/pull/87#discussion_r672428423) for explanation.